src/main/javascript/service/main-menu-service.js

```diff
@@ -97,7 +97,7 @@ function Menu(menuId, options) {
             } else {
                 return [group, group.getEntries()];
             }
-        })]);
+        })], !!_.flattenDeep);
```

Comment:
why would you flatten it deep only if the flattenDeep function is available and not otherwise???

this results in two different outputs depending on the lodash version, which just cannot be the intention

or am i missing somehting